src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/VertexFace.java

```diff
@@ -0,0 +1,48 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+/**
+ * class that describes structure of vertex
+ */
+public class VertexFace {
+    private int index = -1;
+    private VertexGeometry vertexG = null;
```

Comment:
Geometry and Texture vertex can be represented as Vector3.

Reply:
Already done